app/assets/stylesheets/article-show.scss

```diff
@@ -315,6 +316,8 @@ header{
       &.anchor{
         padding-top: 50px;
         margin-top: -50px;
+        display: block;
+        visibility: hidden;
```

Comment:
I'm okay with using `visibility: hidden` since the anchor is not supposed to be read by screen readers.

Reply:
👌